### PR TITLE
POST DATA using php

### PR DESCRIPTION
**Pull request by saurabh2812**

### CODE REVIEW

File: CRUDTask/User.php [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
+    <title>User</title>
+
+    <script>
+       var stateObject = {
```
Comment (avinash-yudiz) [inline CRUDTask/User.php:54]: get countries, states, cities data from database and use Jquery

File: CRUDTask/edit.php [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                <select name="country" class="col-md-3">
+                  <option value="">Country</option>
+                   <option value="Austalia" <?php if ($country == "Austalia") echo 'selected'; ?>>Austalia</option>
+                   <option value="Canada" <?php if ($country == "Canada") echo 'selected'; ?>>Canada</option>
+                   <option value="Argentina" <?php if ($country == "Argentina") echo 'selected'; ?>>Argentina</option>
+                   <option value="France" <?php if ($country == "France") echo 'selected'; ?>>France</option>
+                   <option value="India" <?php if ($country == "India") echo 'selected'; ?>>India</option>
+                
+                </select>
+            </div><br>
```
Comment (avinash-yudiz) [inline CRUDTask/edit.php:119]: get the data from database and show the using ajax, jquery

File: CRUDTask/User.php (hunk 1 above)
Comment (saurabh2812) [reply CRUDTask/User.php:54]: yes sir

File: CRUDTask/UserDisplay.php [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+              <th scope="col">Password</th>
+              <th scope="col">Profile</th>
+              <th scope="col">ASM</th>
+              <th scope="col">Opration</th>
+            </tr>
+      </thead>
+      <tbody >
+
+         <?php 
+            $sql="SELECT * FROM `ueser_add` ";
```
[the remark this answers is not in the document]
Comment (saurabh2812) [reply CRUDTask/UserDisplay.php:42]: yes sir

**Pull request merged**